site/assets/js/color-modes/index.js

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Color mode toggler for Bootstrap's docs (https://getbootstrap.com/)
+ * Copyright 2011-2022 The Bootstrap Authors
+ * Licensed under the Creative Commons Attribution 3.0 Unported License.
```

Comment:
In order to have the same header for all of our JS files.

```suggestion
 * Licensed under the Creative Commons Attribution 3.0 Unported License.
 * For details, see https://creativecommons.org/licenses/by/3.0/.
```

Reply:
Going to streamline the licenses across the project for v5.3.0